src/Form.js

```diff
@@ -0,0 +1,37 @@
+// Form component
+import { useState } from "react";
+
+
+const Form = () => {
+
+   const [ cityChoice, setCityChoice ] = useState("");
+
+   const handleCityChange = (event) => {
+      setCityChoice(event.target.value);
+   }
+
+   const handleSubmit = (event) => {
+      event.preventDefault();
+   }
+
+   return (
+      <section>
+         <h2>The forecast calls for...</h2>
+         <form onSubmit = {handleSubmit}>
+            <label htmlFor="city">Search by city:</label>
+            <input 
+               placeholder="Toronto"
+               required
+               type="text" 
+               id="city" 
+               name="city" 
+               onChange = {handleCityChange} 
+               value = {cityChoice} 
+            />
+         <button type="submit">Search</button>
```

Comment:
I thought we were giving the users a choice of which type of weather they want it to be displayed. (current, hourly, 10 days)
we can either do select or radio.
let me know what you think ^^
I don't mind just displaying all three~